### ISSUE TITLE
Can't access Identity or Profile APIs

### ISSUE THREAD

**Issue by slishak** [microsoft/azure-devops-python-api#188]
I am using the core and git clients and they work perfectly. However, I can't find a way to get the identity of the current signed in user.

If I try and use the profile client, I get messages like this:
```python
profile_client.get_profile('me')
```
```ValueError: API resource location f83735dc-483f-4238-a291-d45f6080a9af is not registered on https://XXX.visualstudio.com.```

Or with the identity client:
```python
identity_client.get_self()
```
```ValueError: API resource location 4bb02b5b-c120-4be2-b68e-21f7c50a4b82 is not registered on https://XXX.vssps.visualstudio.com.```

However, if I manually make the same request in the browser, it works fine (for example https://app.vssps.visualstudio.com/_apis/profile/profiles/me?api-version=5.0).

It seems to only be requests related to the current users's profile which don't resolve. The following request goes through successfully:
```python
identity_client.list_groups()
```

The client is authenticated with OAuth2 with scope `vso.code_full vso.code_status vso.graph vso.identity vso.authorization_grant`.

**Comment by tedchamb** [comment]
@slishak I will investigate.  In the meantime, try this alternate approach to getting the information:

location_client = connection.clients_v5_0.get_location_client()
connection_data = location_client.get_connection_data()
authenticated_user = connection_data.authenticated_user
pprint.pprint(authenticated_user.__dict__)

**Comment by slishak** [comment]
Thanks very much, that snippet of code is exactly what I needed!

**Comment by tedchamb** [comment]
Partial fix for Profile issue: #212

**Comment by tedchamb** [comment]
The profile client talks to the user service, which at this point does not allow PATs.  You would need to use an AAD or MSA credentials to talk to directly to the user service.  Fix #212 is also required.
